tests/test_database_types.py

```diff
@@ -2,33 +2,117 @@
 import unittest
 import time
 import logging
+import re
+import math
+import datetime
 from decimal import Decimal
+from prestodb.exceptions import PrestoUserError
```

Comment:
I think we should avoid this kind of imports.

Instead you can use `import_presto()` from `databases/presto.py` where it's needed.

If that's too clumsy, we can store "prestodb" in the Presto insntance, and use it as `db.prestodb.exceptions.PrestoUserError`.